finetune/config.py

```diff
@@ -206,6 +203,7 @@ def get_default_config():
         eval_acc=False,
         save_dtype=None,
         regression_loss="L2",
+        viable_edges=None,
```

Comment:
we should talk about what pattern we follow for model specific params, because continuously adding to this config object seems kinda suspect